server/routes/aws/cloudwatch_routes.py

```
@@ -0,0 +1,287 @@
+    if not user_id:
+        logger.warning("[CLOUDWATCH] Webhook received without user_id")
+        return jsonify({"error": "user_id is required"}), 400
+
+    if not validate_user_exists(user_id):
+        return jsonify({"error": "Unknown user"}), 404
+
+    # SNS sends JSON; parse body regardless of Content-Type header
+    body = request.get_data(as_text=True)
+    try:
+        sns_message = json.loads(body)
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)